TESTS/scripts/jinja_env/templates/declare_ram_tensor.cpp

```diff
@@ -0,0 +1 @@
+Tensor {{tensor_name}} = new RamTensor({ {%for s in shape%}{{ s }}{{"," if not loop.last}}{%endfor%} }, {{TENSOR_TYPE_MAP[tensor_type_str]}});
```

Comment:
should probably group these into one object, so `{{ tensor.name }}`, `{{ tensor.shape }}`, etc. It's just a little clearer

Reply:
Right, it's temporary I'd say.
Maybe I'll write another Backend for this if I have time.